diff --git a/src/fix_ave_time.h b/src/fix_ave_time.h
index 7e16b0e..a01df02 100644
--- a/src/fix_ave_time.h
+++ b/src/fix_ave_time.h
@@ -36,7 +36,6 @@ class FixAveTime : public Fix {
   double compute_vector(int) override;
   double compute_array(int, int) override;
 
- private:
   struct value_t {
     int which;       // type of data: COMPUTE, FIX, VARIABLE
     int argindex;    // 1-based index if data is vector, else 0
@@ -51,6 +50,14 @@ class FixAveTime : public Fix {
       int v;
     } val;
   };
+
+  bigint nextvalid();
+  int getmode() const { return mode; }
+  int getnvalues() const { return nvalues; }
+  int getnrows() const { return nrows; }
+  const std::vector<value_t> &getValues() const { return values; }
+
+ private:
   std::vector<value_t> values;
 
   int nvalues, nrepeat, nfreq, irepeat;
@@ -84,7 +91,6 @@ class FixAveTime : public Fix {
   void invoke_vector(bigint);
   void options(int, int, char **);
   void allocate_arrays();
-  bigint nextvalid();
 };
 }    // namespace LAMMPS_NS
 #endif
diff --git a/src/input.h b/src/input.h
index 949a70d..14b7ffc 100644
--- a/src/input.h
+++ b/src/input.h
@@ -44,11 +44,10 @@ class Input : protected Pointers {
   void write_echo(const std::string &);                    // send text to active echo file pointers
   int get_jump_skip() const { return jump_skip; }
 
- protected:
   int echo_screen;    // 0 = no, 1 = yes
   int echo_log;       // 0 = no, 1 = yes
 
- protected:
+
   int me;                           // proc ID
   int maxarg;                       // max # of args in arg
   char *line, *copy, *work;         // input line & copy and work string
diff --git a/src/modify.cpp b/src/modify.cpp
index 5a125d7..b79f3d4 100644
--- a/src/modify.cpp
+++ b/src/modify.cpp
@@ -830,7 +830,7 @@ Fix *Modify::add_fix(int narg, char **arg, int trysuffix)
   // clang-format off
   const char *exceptions[] =
     {"GPU", "OMP", "INTEL", "property/atom", "cmap", "cmap3", "rx", "deprecated", "STORE/KIM",
-     "amoeba/pitorsion", "amoeba/bitorsion", "DUMMY", nullptr};
+     "amoeba/pitorsion", "amoeba/bitorsion", "DUMMY", "atomify", nullptr};
   // clang-format on
 
   if (domain->box_exist == 0) {
diff --git a/src/neigh_request.h b/src/neigh_request.h
index caa9e05..3dc69e5 100644
--- a/src/neigh_request.h
+++ b/src/neigh_request.h
@@ -28,6 +28,7 @@ class NeighRequest : protected Pointers {
   friend class NPairSkipIntel;
   friend class NPairSkipTrimIntel;
   friend class FixIntel;
+  friend class FixAtomify;
 
  public:
   enum { REGULAR, INTRA, INTER };
